File: CS/CellStyleExample/Order.cs
using DevExpress.Mvvm;

namespace CellStyleExample {
    public class Order : BindableBase {


        protected int _OrderId;
        public int OrderId
        {
            get { return this._OrderId; }
            set { this.SetProperty(ref this._OrderId, value, "OrderId"); }
        }

        protected string _CustomerName;
        public string CustomerName
        {
            get { return this._CustomerName; }
            set { this.SetProperty(ref this._CustomerName, value, "CustomerName"); }
        }


        protected int _Quantity;
        public int Quantity
        {
            get { return this._Quantity; }
            set { this.SetProperty(ref this._Quantity, value, "Quantity"); }
        }


        protected double _Price;
        public double Price
        {
            get { return this._Price; }
            set { this.SetProperty(ref this._Price, value, "Price"); }
        }

    }
}

File: VB/CellStyleExample/Order.vb
Imports DevExpress.Mvvm

Namespace CellStyleExample
    Public Class Order
        Inherits BindableBase


        Protected _OrderId As Integer
        Public Property OrderId() As Integer
            Get
                Return Me._OrderId
            End Get
            Set(ByVal value As Integer)
                Me.SetProperty(Me._OrderId, value, "OrderId")
            End Set
        End Property

        Protected _CustomerName As String
        Public Property CustomerName() As String
            Get
                Return Me._CustomerName
            End Get
            Set(ByVal value As String)
                Me.SetProperty(Me._CustomerName, value, "CustomerName")
            End Set
        End Property


        Protected _Quantity As Integer
        Public Property Quantity() As Integer
            Get
                Return Me._Quantity
            End Get
            Set(ByVal value As Integer)
                Me.SetProperty(Me._Quantity, value, "Quantity")
            End Set
        End Property


        Protected _Price As Double
        Public Property Price() As Double
            Get
                Return Me._Price
            End Get
            Set(ByVal value As Double)
                Me.SetProperty(Me._Price, value, "Price")
            End Set
        End Property

    End Class
End Namespace
